hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestRemoteRestore.java

```diff
@@ -72,4 +80,46 @@ public void testFullRestoreRemote() throws Exception {
     TEST_UTIL.deleteTable(table1_restore);
     hba.close();
   }
+
+  @Test
+  public void testFullRestoreRemoteWithAlternateRestoreOutputDir() throws Exception {
+    LOG.info("test remote full backup on a single table with alternate restore output dir");
+    String backupId =
+      backupTables(BackupType.FULL, toList(table1.getNameAsString()), BACKUP_REMOTE_ROOT_DIR);
+    LOG.info("backup complete");
+    TableName[] tableset = new TableName[] { table1 };
+    TableName[] tablemap = new TableName[] { table1_restore };
+
+    HBaseTestingUtil mrTestUtil = new HBaseTestingUtil();
```

Comment:
can you add a comment here about why we need a 3rd cluster. or add a javadoc to the test method explaining the scenario